src/main.ts

```diff
@@ -1,7 +1,20 @@
 import { App } from './app';
+import express from 'express';
+import { userCreateRouter } from './routes/Users/userCreateRoute';
+import { userGetOneRoute } from './routes/Users/userGetOneRoute';
+import { userAuthorsGetRoute } from './routes/Users/userAuthorsGetRoute';
+import { userDeleteRoute } from './routes/Users/userDeleteRoute';
+import { userUpdateRoute } from './routes/Users/userUpdateRoute';
 
 const app = new App();
 
+app.app.use(express.json());
+app.app.use('/user', userCreateRouter);
+app.app.use('/user', userGetOneRoute);
+app.app.use('/user', userAuthorsGetRoute);
+app.app.use('/user', userDeleteRoute);
+app.app.use('/user', userUpdateRoute);
```

Comment:
To avoid a lot of exporting in main file I would recommend create something like `user-route.ts` file, where all users routes are exported, `index.ts` where all routes are exported and connected and separate directory for all routes files